torch_xla/utils/gcsfs.py

```diff
@@ -212,7 +212,7 @@ def remove(path):
   bucket.delete_blob(bpath)
 
 
-def write(path, content):
+def write(path, content, service_account_filename=None):
```

Comment:
No please. Bring down global configuration like this is a PITA.
```Shell
export GOOGLE_APPLICATION_CREDENTIAL=...
```

Reply:
I switched to use env var - we'll see if this works for the Google use case. We run the daily tests in one GCS project but I want to write metrics history to a pytorch-specific GCS project. The test infra does some read/writes to the team-wide GCS project but hopefully those are all done by the time the PyTorch-specific code (including this) kicks in